### ISSUE TITLE
Duplicati 2 try to compact small files that are just compress with 7zip

### ISSUE THREAD

**Issue by kgb34** [duplicati/duplicati#1170]
Hello,
I use duplicati 2 with 7z compress option ans 100mb block size on local drive for about 30Go and 150000 files.
With 7z some block files are small (1.5 to 10mb) but they really contain 1024 blocks of 100ko ...
So duplicati consider them as small files and try to compact them ...
so i use the --no-auto-compact switch but i think there is a problem

Thanks for your answer
##

**Comment by ts678** [comment]
[2.0.7.100_canary_2023-12-27](https://github.com/duplicati/duplicati/releases/tag/v2.0.7.100-2.0.7.100_canary_2023-12-27) didn't immediately make small files, but also didn't work. The job log now logs a general Warning:

> 2024-02-17 10:03:11 -05 - [Warning-Duplicati.Library.Main.Controller-7zModuleHasIssues]: The 7z compression module has known issues and should only be used for experimental purposes

[Disable or fix 7z/LZMA2 #3054](https://github.com/duplicati/duplicati/issues/3054) added that, and as after several more years, the question of that closed topic still seems relevant.

My job log also had an Error, and About --> Show log gave me a better look. Although I won't likely file an issue, that error was:

```
System.IO.FileNotFoundException: The given file is not part of this archive
File name: 'fileset'
   at Duplicati.Library.Compression.SevenZipCompression.OpenRead(String file)
   at Duplicati.Library.Main.Volumes.VolumeReaderBase.ReadFileset()
   at Duplicati.Library.Main.Operation.Backup.SpillCollectorProcess.<>c__DisplayClass0_0.<<Run>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CoCoL.AutomationExtensions.<RunTask>d__10`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Duplicati.Library.Main.Operation.BackupHandler.<RunMainOperation>d__13.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Duplicati.Library.Main.Operation.BackupHandler.<RunAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at CoCoL.ChannelExtensions.WaitForTaskOrThrow(Task task)
   at Duplicati.Library.Main.Controller.<>c__DisplayClass14_0.<Backup>b__0(BackupResults result)
   at Duplicati.Library.Main.Controller.RunAction[T](T result, String[]& paths, IFilter& filter, Action`1 method)
   at Duplicati.Library.Main.Controller.Backup(String[] inputsources, IFilter filter)
   at Duplicati.Server.Runner.Run(IRunnerData data, Boolean fromQueue)
```

Actual 7-Zip can't even open the supposedly `.7z` files in the backup, or the one in `Temp`. `Test archive` gives a Headers Error.

**Comment by kenkendk** [comment]
Closing this as 7z is now removed.
